shaka/src/public/ShakaPlayerView.mm

```diff
@@ -175,8 +183,8 @@ - (void)renderLoop {
     shaka::ShakaRect<uint32_t> dest_bounds = {
         0,
         0,
-        static_cast<uint32_t>(self.bounds.size.width),
-        static_cast<uint32_t>(self.bounds.size.height),
+        static_cast<uint32_t>(self.superview.bounds.size.width),
+        static_cast<uint32_t>(self.superview.bounds.size.height),
```

Comment:
I don't think this is correct.  `self` refers to the whole video view, which may be only a part of the `superview`.  The app may position or size the video in part of the parent.  How are the returned values wrong?  Maybe it is because the view hasn't been updated yet?  Maybe there is a different event once this view has been resized within the new orientation?  What about using `layoutSubviews`?

Reply:
As I learned, after device orientation changed, bounds should be recalculated and frame stills the same. But in this case, bounds were refering to same value as frame after orientantion change. So eg. after change from portrait to landscape, height was bigger then width (height > width) which in landscape mode is incorrect. Correct dimensions I get from superview OR from [UIScreen mainScreen].bounds.size;

Yes, I tought to that maybe its wrong event so I tried it in layoutSubviews, but result was same wrong.

Reply:
Superview removed in last commit and solution is in last commnet